require_node.py

```diff
@@ -30,7 +30,8 @@ def write(index):
 
             region_to_insert = self.view.sel()[0]
 
-            line_is_empty = self.view.lines(region_to_insert)[0].empty()
+            line = self.view.substr(self.view.lines(region_to_insert)[0])
```

Comment:
What's the purpose of the change in these two lines?

Sorry, I am not very familiar with python and the sublime api